docs: Add Cloud Logging Dashboards for metrics (#751)
kyma-project/telemetry-manager: docs/user/integration/sap-cloud-logging/README.md

SAP Cloud Logging provides an extensive set of dashboards under `OpenSearch Dashboards > Dashboard`, which give insights in traffic and application logs. These dashboards are prefixed with `Kyma_`, and are based on both kinds of [log ingestion](#ship-logs-to-sap-cloud-logging): application and access logs.

Additionally, you can view distributed traces under `OpenSearch Plugins > Observability`.

> For visualization of the metrics generated by the inputs of a MetricPipeline, the following Dashboards are available for a manual upload. In the SAP Cloud Logging Dashboard navigate to **Stack Managament > Saved Objects** and click the **Import** button to upload the files listed below:

Reviewer: ```suggestion
To view the metrics generated by the inputs of a MetricPipeline in a Dashboard, you can manually upload them. In the SAP Cloud Logging Dashboard, go to **Stack Management > Saved Objects** and import the following files:
```

| Category | File | Description |
| -- | -- | -- |

> | Runtime Metrics | [Kyma Container Metrics](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/dashboard-runtime.ndjson) | Visualizes the container and pod related metrics collected by the MetricPipeline `runtime` input.|

Reviewer: ```suggestion
| Runtime Metrics | [Kyma Container Metrics](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/dashboard-runtime.ndjson) | Visualizes the container- and Pod-related metrics collected by the MetricPipeline `runtime` input.|
```

> | Istio Metrics | [Kyma Istio Service Metrics](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/dashboard-istio.ndjson) | Visualizes the Istio metrics of pods having an active Istio sidecar injection. The metrics are collected by the MetricPipeline `istio` input. |

Reviewer: ```suggestion
| Istio Metrics | [Kyma Istio Service Metrics](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/dashboard-istio.ndjson) | Visualizes the Istio metrics of Pods that have an active Istio sidecar injection. The metrics are collected by the MetricPipeline `istio` input. |
```
